examples/atlas-api-key/create-and-assign-pak-seperately/main.tf

```diff
@@ -0,0 +1,32 @@
+
+# Create Programmatic API Key
+resource "mongodbatlas_api_key" "orgKey1" {
+  description = "orgKey2"
+  org_id      = var.org_id
+  role_names  = ["ORG_OWNER"]
+}
+
+# Assign Newly Created Programmatic API Key to Project 
```

Comment:
Should we provide a warning that this way is being deprecated in the near future?

Reply:
added!